UPDATING.md

```diff
@@ -89,6 +89,11 @@ installing Airflow, example for Python 3.8 and Airflow 2.1.2:
 pip install -U "apache-airflow[pandas]==2.1.2" \
   --constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.1.2/constraints-3.8.txt"
 ```
+### `none_failed_or_skipped` trigger rule has been deprecated
```

Comment:
```suggestion

### `none_failed_or_skipped` trigger rule has been deprecated
```

Reply:
Should fix failing static check- https://github.com/apache/airflow/pull/17683/checks?check_run_id=3372171479#step:10:242